infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile

```diff
@@ -31,21 +31,13 @@ gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub \n\
 " > /etc/yum.repos.d/google-chrome.repo
 
 RUN yum update -y
-
 RUN yum install -y java-1.8.0-openjdk git google-chrome-stable bind-utils net-tools
-
-RUN curl -LO https://nodejs.org/dist/v12.16.3/node-v12.16.3-linux-x64.tar.xz
```

Comment:
Doesn't getting rid of this cause the subsequent `npm install` steps to fail?

Reply:
`curl -LO` only downloads the file, there is nothing done with it (as far as I can tell) after the download.

Reply:
It's un-tarred into `/usr/local` on [line 37 of your branch](https://github.com/apache/trafficcontrol/pull/4682/files#diff-25b7184084d757a7bbf533c72127e049R37) - which means that as of now the integration container fails to actually build.

Reply:
That is interesting, it was definitely still working when I tested. Must have had something cached.